packages/app/background-jobs-common/src/background-job-processor/factories/sanitizedRedisClientFactory.ts

```diff
@@ -0,0 +1,6 @@
+import type { RedisConfig } from '@lokalise/node-core'
+import Redis from 'ioredis'
+import { sanitizeRedisConfig } from '../utils'
+
+export const createSanitizedRedisClient = (redisConfig: RedisConfig): Redis =>
```

Comment:
In some places we will still want to operate on Redis instance, not on config, while taking advantage of sanitization step. Therefore I think it's useful to have function that allows to create such instance. Let me know if you disagree

Reply:
we don't have a factory per se here, just a one-liner wrapper.

what is the added value on top of the `sanitizeRedisConfig` method? Can't user apply `sanitizeRedisConfig` on their own when creating a new instance?

I'm not against having it, primarily not convinced that it deserves a separate file. can be just part of utils

Reply:
Initially I added it as a part of utils, but those are internal and not exported, so finally decided to put public one in separate file. I noticed in `UPGRADING.md` that with recent version `sanitizeRedisConfig` was explicitly excluded from public ones, not sure why. 
But I can put it back to utils and export that one function if you think it's better, no strong opinion from my side

Reply:
Regarding `sanitizeRedisConfig` - sure, it could be exported and used as well, I was just thinking that if it's always used just for `new Redis(sanitizedConfig)`, we can already have it wrapped. Also no strong preference :)

Reply:
I would be in favour of exporting the sanitization method, but also I'm not against having a separate method for instantiation, we can have both

Reply:
Moved it to utils and exported those 2 metrics